src/common/colourdata.cpp

```diff
@@ -102,6 +107,17 @@ bool wxColourData::FromString(const wxString& str)
         else
             success = m_custColours[i].Set(token);
     }
+
+    token = tokenizer.GetNextToken();
+    if ( token.empty() )
```

Comment:
This is very minor but why not just do `m_showOpacity = token == "1";`, it seems like it would have exactly the same effect (except it would also work with strings in invalid format, e.g. using "2").